src/reference/sectionC_interpretation.inc

```diff
@@ -167,33 +167,38 @@ Interpretation of ``units`` elements
 
 .. specC_irreducible_units:
 
-2. For the purposes of this specification, the "irreducible units" of a model SHALL consist of 1) the units defined in a model that are not defined in terms of other units (i.e. the set of :code:`units` elements in the :ref:`CellML model<specA_cellml_model>` which have no :code:`unit` child elements), and 2) built-in irreducible units (those built-in units with "-" in the "Unit Reduction" column of the :ref:`Built-in units<table_built_in_units>` table) referenced by variables or other units in the model.
+2. For the purposes of this specification, the "irreducible units" of a model SHALL consist of:
+
+   1. The units defined in a model that are not defined in terms of other units (i.e. the set of :code:`units` elements in the :ref:`CellML model<specA_cellml_model>` which have no :code:`unit` child elements).
+
+   2. The built-in irreducible units (those built-in units with "-" in the "Unit Reduction" column of the :ref:`Built-in units<table_built_in_units>` table) referenced by variables or other units in the model.
 
 .. marker_interpretation_of_units_2
 
 3. The "unit reduction" is a conceptual property of :code:`units` elements.
-   It consists of a set of tuples where each tuple is composed of a) a unit name and b) a real-valued exponent.
-   Tuples SHALL be determined as follows:
+   It consists of a set of tuples where each tuple is composed of a "unit name" and a real-valued "exponent".
+   The set of tuples SHALL be determined as follows:
 
    1. If the :code:`units` element has no :code:`unit` child elements, then the set of tuples SHALL have a single member, which SHALL consist of the name of the :code:`units` element and the exponent 1.0.
 
    2. If the :code:`units` element has one or more :code:`unit` child elements, then the set of tuples SHALL consist of the entire collection of tuples given by all :code:`unit` child elements.
+
       Tuples for each :code:`unit` child element SHALL be determined as follows:
 
-      1. Where the :ref:`units reference<specC_units_reference>` of the :code:`unit` child element is to a single unit which is an irreducible unit, then the set of tuples SHALL have a single member, which SHALL consist of the name of the irreducible unit being referenced and the exponent 1.0.
+      1. If the :ref:`units reference<specC_units_reference>` of the :code:`unit` child element is to a single unit which is an irreducible unit, then the set of tuples SHALL have a single member, which SHALL consist of the name of the irreducible unit being referenced and the exponent 1.0.
```

Comment:
Gah, this is why I so dislike "units" and "unit" terms ... how can we make it clear that the bold below is talking about physical units (singular) rather than the unit item? 

If the :ref:`units reference<specC_units_reference>` of the :code:`unit` child element is **to a single unit which is an irreducible unit**, then the set of tuples SHALL have a single member, which SHALL consist of the name of the irreducible unit being referenced and the exponent 1.0.

Reply:
Yeah, I know.  If everyone reads it with as much attention to font as we do, we'll all be ok ... ;)

Reply:
FWIW, I am ok with the current text.

Reply:
I don't want to introduce new terminology either

Reply:
I'm gonna hit resolve on these are they're not about the changes in the PR!